tests/web/frontend/conftest.py

```diff
@@ -59,6 +65,32 @@ def run_uvicorn_web_server(
 ###############################
 
 
+@pytest.fixture(scope="module")
+def aiida_profile(aiida_config, aiida_profile_factory):
```

Comment:
I changed back to aiida_profile as this will overwrite the aiida_profile in the main conftest.py